### DISCUSSION TITLE
FlareSolverr through vpn Windows

### DISCUSSION THREAD

**Discussion by Linouxs** [FlareSolverr/FlareSolverr#740] [Q&A]
Hi, Love it that the binary for Windows is added in the latest update. I would like to run FlareSolverr through the same vpn as Prowlarr (at the same pc) with split tunneling. But the problem is that I can't add the executable from Chromium to the vpn (C:\users\user\appdata\local\temp\\...\chrome.exe) because the location of the executable changes sometimes. I can't disable split tunneling because Plex and other services are running at the same device. Would it be possible to use a fixed location for the Chromium executable?

**Comment by ngosang** [comment]
https://github.com/FlareSolverr/FlareSolverr/releases/tag/v3.1.1
